wpa_supplicant/ctrl_iface_zephyr.c

```diff
@@ -8,6 +8,123 @@
 
 #include "ctrl_iface_zephyr.h"
 
+
+static int wpa_supplicant_ctrl_iface_attach(struct wpa_ctrl_dst **head, int sock)
```

Comment:
Can we please add `mon` prefix to clearly indicate that these are to monitor events? `wpa_supplicant_ctrl_iface_mon_attach`